### PR TITLE
[T2A3][W09-A2]Lee Pei Xuan

### PR DESCRIPTION
**Pull request by pyxyyy**

### CODE REVIEW

**Comment by pyxyyy** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -208,7 +208,46 @@
...
     public static void main(String[] args) {
         showWelcomeMessage();
+        
+        /* original code before SLAP reduction
         processProgramArgs(args);
+        */
+
+        /** T2A3 Exercise 1
+         * After SLAP reduction
+         */
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/AddressBook.java:218]: Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -208,7 +208,46 @@
...
+            } else {
+                if (args.length == 1) {
+                    setupGivenFileForStorage(args[0]);
+                } else {
+                    if (args.length == 0) {
+                        setupDefaultFileForStorage();
+                    }
+                }
+            }
+        }
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/AddressBook.java:248]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

**Comment by jeffryhartanto** [comment]
@pyxyyy  Some comments added. Please close the PR after reading comments.

Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

Also, you don't need to revert back to original code if its unnecessary to do so.

**Pull request closed without merging**